config/openshift/cryostat-quickstart-jmx.yaml

```diff
@@ -1,29 +1,30 @@
-  description: Configure your Java applications on OpenShift to be profiled and monitored using the Red Hat build of Cryostat.
+  description: Configure your JMX-based Java applications on OpenShift to be profiled and monitored using the Red Hat build of Cryostat.
   durationMinutes: 10
-  displayName: Configure Java applications with the Red Hat build of Cryostat
+  displayName: Configure JMX-based Java applications with the Red Hat build of Cryostat
   introduction: >-
     Red Hat build of Cryostat is a container-native Java application based on JDK Flight Recorder (JFR). You can use it to monitor the performance of the Java Virtual Machine (JVM) for containerized workloads that run on an OpenShift cluster.
 
     - To enable Cryostat to gather, store, and analyze Java Flight Recorder (JFR) data about target applications that run on Java Virtual Machine (JVM)s, you must configure the applications so that Cryostat can detect and connect to them.
 
-    - This requires you to configure your Java applications to allow Java Management Extensions (JMX) connections and use an OpenShift Service for detection and JMX for connectivity.
+    - You can use Red Hat build of Cryostat Operator to automatically configure your target application to use the Cryostat Agent to register with a specific Cryostat instance and provide HTTP-based connectivity by following the **Auto-configure Java applications with the Red Hat build of Cryostat** quick start.
```

Comment:
Now that we have 2 quickstart and this one reference the `configuring-cryostat-autoconfig` quickstart. What do we think of linking them?

https://docs.openshift.com/container-platform/4.17/web_console/creating-quick-start-tutorials.html#linking-to-other-quick-starts_creating-quick-start-tutorials

Reply:
Added in the latest commit, though for some reason the console seems to double up the links:

![image](https://github.com/user-attachments/assets/b54f9daa-df1d-4150-a596-0c282d115c34)

Reply:
Kinda funny. If you open the cryostat quickstart on the side panel, navigate away (i.e. go to any other console tab) and do a page refresh, the link looks fine. So, it might be a console bug :D 

![image](https://github.com/user-attachments/assets/e4a02812-0b7c-4af2-ab38-9c1d1845580e)